setup.py

```diff
@@ -79,6 +79,7 @@
             "hydra-zen >= 0.7.0",
             "xarray >= 0.19.0",
             "matplotlib >= 3.3",
+            "pooch >= 1.6.0",
```

Comment:
Should we put this in [experiments/setup.py](https://github.com/mit-ll-responsible-ai/responsible-ai-toolbox/blob/main/experiments/setup.py) instead?
Or do you anticipate there might be future cases where we'll need to use pooch with the main toolbox?

Reply:
Although I'm also wondering if we could move some of what is currently in `experiments/src` into the main toolbox (maybe `mushin`?) so that we can access it more easily for the tutorials (right now we [direct users to clone the repo](https://mit-ll-responsible-ai.github.io/responsible-ai-toolbox/tutorials/CIFAR10-Adversarial-Perturbations.html#Installing-rai_experiments) and install `rai_experiments` to run the tutorials which is cumbersome)

Reply:
I'd prefer to keep `rai_experiments` separate, but perhaps we can make it pip-installable!